warehouse/validations/validate.go

```diff
@@ -116,18 +116,34 @@ func (ct *CTHandleT) verifyingObjectStorage() (err error) {
 	return
 }
 
+func (ct *CTHandleT) manageTunnellingSecrets(config map[string]interface{}) error {
+	if !warehouseutils.ReadAsBool("useSSH", config) {
+		return nil
+	}
+
+	sshKeyId, ok := ct.warehouse.Destination.Config["sshKeyId"]
+	if !ok {
+		return fmt.Errorf("missing sshKeyId in validation payload")
+	}
+
+	keys, err := ct.CPClient.GetSSHKeys(context.TODO(), sshKeyId.(string))
```

Comment:
```suggestion
    	sshKey, ok := sshKeyId.(string)
	if !ok {
		return fmt.Errorf("invalid sshKeyId")
	}

	keys, err := ct.CPClient.GetSSHKeys(context.TODO(), sshKey)
```

Should we have an extra check here whether sskKeyID is of the proper format?

Reply:
Having a different datatype would be a very big deviation from the standard. In case that happens, it's better to let us know asap, so panic might help us. Wdyt ?

Reply:
I prefer @achettyiitr suggestion, to be honest. 

Given the schema of the payload is not in our control and we don't validate against the contract when we receive the request in our system. 

I don't see it as an unlikely scenario a small change in the control plane, encoding `sshKeyId` as int and then having warehouse panic when someone tries to setup an ssh destination.